packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFHTTPCookieStoreHostApiTests.m

```diff
@@ -14,15 +14,15 @@ @interface FWFHTTPCookieStoreHostApiTests : XCTestCase
 @implementation FWFHTTPCookieStoreHostApiTests
 - (void)testCreateFromWebsiteDataStoreWithIdentifier API_AVAILABLE(ios(11.0)) {
   FWFInstanceManager *instanceManager = [[FWFInstanceManager alloc] init];
-  FWFHTTPCookieStoreHostApiImpl *hostApi =
+  FWFHTTPCookieStoreHostApiImpl *hostAPI =
       [[FWFHTTPCookieStoreHostApiImpl alloc] initWithInstanceManager:instanceManager];
 
   WKWebsiteDataStore *mockDataStore = OCMClassMock([WKWebsiteDataStore class]);
   OCMStub([mockDataStore httpCookieStore]).andReturn(OCMClassMock([WKHTTPCookieStore class]));
-  [instanceManager addInstance:mockDataStore withIdentifier:0];
+  [instanceManager addInstanceCreatedFromDart:mockDataStore withIdentifier:0];
```

Comment:
I just noticed that the PR description says `addDartCreatedInstance`, but this is `addInstanceCreatedFromDart`. The former would be better, because it puts the word "instance" right before the instance parameter, per Apple naming guidelines.

Reply:
I commented about this in the other PR. I'll update the name in both!